app/model/JsVars.scala

```diff
@@ -3,16 +3,33 @@ package model
 import play.api.libs.json._
 
 object JsVars {
-  def default = JsVars(userIsSignedIn = false, ignorePageLoadTracking = false)
+  def default = JsVars(
+    userIsSignedIn = false,
+    ignorePageLoadTracking = false,
+    stripePublicKey = None
+  )
 
   implicit val jsVarsWrites = new Writes[JsVars] {
-    def writes(jsVars: JsVars) = Json.obj(
-      "user" -> Json.obj(
-        "isSignedIn" -> jsVars.userIsSignedIn,
-        "ignorePageLoadTracking" -> jsVars.ignorePageLoadTracking
+    def writes(jsVars: JsVars) = {
```

Comment:
If we use the `Json.writes[JsVars]` macro, the optional fields don't appear when null. It might be more concise and less confusing to use the standard serialization for `JsVars` and wrap it inside a `user` field at call point.